honeybee_vtk/model.py

```diff
@@ -153,6 +155,15 @@ def cameras(self):
         """List of Camera objects attached to this Model object."""
         return self._cameras
 
+    def get_modeldataset_from_string(self, text: str) -> ModelDataSet:
```

Comment:
We need to document this method. I'm not sure what does it do?

Reply:
Complied.